src/app/time-series-menus/climatology-graph.component.html

```diff
@@ -0,0 +1,89 @@
+<div id="ClimatologyGraph">
+  <div id="climatology-print-section">
```

Comment:
for the Global Surface Temperature the title gets too long and is covered by the closing button. Can we just reduce the font just little to fit for longer titles. Also the legend and title get too close. Can we just move the Legend a bit low. I will send you screenshot